sdk/appconfiguration/ci.yml

```diff
@@ -12,23 +12,23 @@ trigger:
       - sdk/appconfiguration/azure-data-appconfiguration/
       - sdk/appconfiguration/azure-data-appconfiguration-perf/
       - sdk/appconfiguration/azure-resourcemanager-appconfiguration/
-      - sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/
-      - sdk/appconfiguration/azure-spring-cloud-appconfiguration-config-web/
+#      - sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/
```

Comment:
//CC @alzimmermsft @srnagar 
Again, this goes beyond the exclusion that was done in the generate_from_source_pom.py to exclude spring libraries that should probably be being built.